releases/net472/KnownIssues/560016-XBAP Applications targeting .NET 2.0,3.5 select wrong runtime version.md

```diff
@@ -0,0 +1,28 @@
+# [|Known Issue Title|]
+XBAP Applications targeting .NET 2.0/.NET 3.5 select wrong runtime (v4.0 CLR) instead of v2.0 CLR 
```

Comment:
lowercase "a" in applications
.NET --> .NET Framework (2 occurrences)
this can just read, "XBAP applications targeting .NET Framework 2.0/3.5 select the wrong runtime"

Reply:
Fixed